sfcc_cartridge/int_fast_sfra/cartridge/static/default/js/fastProduct.js

```diff
@@ -24,12 +24,12 @@ $(document).ready(function () {
 
     checkoutButton.addEventListener('click', event => {
         Fast.checkout({
-            appId: checkoutButton.getAttribute('app_id'),
+            appId: checkoutButton.getAttribute('fast_app_id'),
             buttonId: event.target.id,
             products: [
                 {
-                    id: $('.product-id').html(),
-                    variantId: $('.product-id').html(),
+                    id: checkoutButton.getAttribute('product_id'),
+                    variantId: checkoutButton.getAttribute('product_id'),
```

Comment:
so variant id and id are the same?

Reply:
Yes, it was same in the original design so i kept as is but with fixed lookup. I can see that product id is updated after variant selection, this might be SFCC feature where variant id = product id always